gef-ui/server/utils/fileUtils.js

```diff
@@ -9,19 +9,23 @@ const {
  * @param {array} files array of files to upload, assumes format from multer
  * @param {string} field field identifier for supplementaryInfo question
  * @param {string} dealId application/deal ID
- * @param {string} token user token
+ * @param {string} userToken user token
  * @param {number} maxFileSize maximum file size to allow (defaults to API limits of 10MB otherwise)
  * @param {string} documentPath full document path (i.e. manualInclusion)
  * @returns array of processed files
  */
-const uploadAndSaveToDeal = async (files, field, dealId, token, user, maxFileSize, documentPath) => {
-  const uploadedFiles = await uploadFile(files, dealId, token, maxFileSize, documentPath);
+const uploadAndSaveToDeal = async (files, field, dealId, userToken, user, maxFileSize, documentPath) => {
```

Comment:
Pass object as argument?

Reply:
I'm going to leave as-is since I've not changed the number of arguments, just to limit the scope of this change